doc/content/getting_started/hpc.md

```diff
@@ -0,0 +1,114 @@
+# Using TMAP8 on INL HPC Systems
+
+!alert! note
+This guide will discuss how to use TMAP8 in an [!ac](INL) [!ac](HPC) environment. To
+install TMAP8 on your own [!ac](HPC) system, you should start by reviewing the
+[MOOSE HPC Cluster instructions](https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html)
+for information on how to install MOOSE (and MOOSE-based applications like TMAP8) and working with
+your system administrator to get the appropriate software dependencies installed. Once this is done,
+proceed with the manual HPC installation instructions below in Step Three.
+!alert-end!
+
+## Step One: Get an INL HPC Account
+
+!style halign=left
+To use TMAP8 on [!ac](INL) [!ac](HPC), you first need an [!ac](HPC) account. Information on direct
+access to [!ac](HPC), or through the web-based HPC OnDemand service, can be found on the
+MOOSE website [INL HPC Services](https://mooseframework.inl.gov/help/inl/index.html) page.
+
+## Step Two: Set Up the Right Environment
+
+!style halign=left
+An optimized, pre-packaged environment is available for use on the Sawtooth and Lemhi supercomputers.
+When logged in, run:
+
+```bash
+module load use.moose moose-dev
+```
+
+and a set of appropriate MOOSE dependency modules will be loaded for use. Note that these modules
++require activation+ each time you log in.
+
+## Step Three: Clone TMAP8
+
+<!-- Re-use the clone instructions from the main getting started instructions here -->
+
+!style halign=left
+!include getting_started/installation.md start=TMAP8 is hosted on end=!alert-end! include-end=True
+
+## Step Four: Build and Test TMAP8
+
+!style halign=left
+To compile TMAP8, first make sure that the [!ac](HPC) environment is activated from Step Two (+and be sure to do this any time that a new HPC Terminal instance is opened+):
+
+```bash
+module load use.moose moose-dev
+```
+
+<!-- Re-use the bulk of the build and test of the main getting started instructions here -->
+
+!include getting_started/installation.md start=Then navigate to the TMAP8 end=TMAP8 can be compiled and tested include-end=True
+
+```bash
+make -j8
+./run_tests -j8
+```
```

Comment:
The suggestion thing in GitHub is struggling with all the '`' and I can't figure out how to make it work, so here it it in text:


```bash
make -j8
```

Once TMAP8 has successfully been compiled, the following command runs the TMAP8 active tests:
```bash
./run_tests -j8
```